externs/es6.js

```diff
@@ -2035,6 +2035,31 @@ function WeakRef(value) {}
  */
 WeakRef.prototype.deref = function() {};
 
+/**
+ * @constructor
+ * @struct
+ * @param {function(HOLDINGS)} cleanupCallback
+ * @template TARGET, HOLDINGS, TOKEN
+ * @nosideeffects
+ * @see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/FinalizationRegistry
+ */
+function FinalizationRegistry(cleanupCallback) {}
+
+/**
+ * @param {TARGET} target
+ * @param {HOLDINGS} holdings
+ * @param {TOKEN=} unregisterToken
+ * @return {void}
+ */
+FinalizationRegistry.prototype.register =
+    function(target, holdings, unregisterToken) {};
```

Comment:
nit: rename parameter to `heldValue` to stay consistent with the spec.

Reply:
My externs _were_ consistent with the spec before [they decided to change the spec](https://github.com/tc39/proposal-weakrefs/issues/113)!  🙄  Fixed.